### ISSUE TITLE
What is the license to this code?

### ISSUE THREAD

**Issue by nicost** [andrewgomella/NikonKsAdapter#1]
Just became aware of this device adapter (just 5 years late I guess;).  What is the license to this code?  If it is an open source compatible license (, i.e. BSD, lGPL, MIT, apache or similar), I would be happy to add this to the Micro-Manager source code repository and ship it in binary form.

**Comment by andrewgomella** [comment]
Good idea.. recently had someone email me to help them get it working so I
imagine at least one person out there would find it useful. Only issue is
it requires some proprietary nikon dll or obj file to run.

BSD. I need to update github to reflect this.

Thanks,
Andrew

**Comment by catalintucureanu** [comment]
You can add at least two persons finding it useful, Andrew. Been using it for about 4 years on two DS-Qi2 cameras on our Nikon microscopes. Had to make some tweaks, since it was not correctly reporting pixel size (micromanager had a fallback mode to a ridiculously small number irrespective of the optical configuration). Will run a diff when I get a bit of time in the following days and submit the code back to you if there is interest in integrating it in the main MM repository. 
Thanks for the great work!

**Comment by andrewgomella** [comment]
That's great to know! Would definitely appreciate you adding the diff. I no
longer have access to qi2 so I have not been updating the code in quite a
while. However there is always the sim mode included in the nikon driver.

Thanks.
